pvr.hts/resources/language/resource.language.en_gb/strings.po

```diff
@@ -161,5 +187,9 @@ msgid "Number of subscriptions to use"
 msgstr ""
 
 msgctxt "#30402"
-msgid "Unused subscription close delay"
+msgid "Unused subscription close delay (secs)"
```

Comment:
Sorry to continue the nitpicking but we should be consistent in how we indicate the unit of measurement. On the connection tab we use `Connection timeout in seconds`, I think we should either change that (and any others) to `Connection timeout (secs)` or change this one. Furthermore I like `(seconds)` better than `(secs)`, but it's not a big deal if it would make the setting name too long.